app/models/user.rb

```diff
@@ -8,6 +8,30 @@ class User < ApplicationRecord
   # @description: Generates the password_digest to ensure it's stored in a secure way
   ###
   has_secure_password
+
+  ###
+  # @description: Rules to use in the password_strength policy
+  ###
+  PASSWORD_VALIDATION_RULES = {
+    ###
+    # @description: Level of deductibility. 18 is the library's default, known as an acceptable level
+    #   of entropy.
+    ###
+    min_entropy: 15,
+    ###
+    # @description: The mimimum acceptable length of the password.
+    ###
+    min_word_length: ENV["MIN_PASSWORD_LENGTH"].to_i || 8,
```

Comment:
I don't think this expression would work as expected. Calling `#to_i` on `nil` or `''` will return `0`, it won't fall back to `8` ever.

Reply:
Good catch!
Changing to `Integer(ENV["MIN_PASSWORD_LENGTH"]) rescue 8`